takecare19/takecare19
# Filter by tags 🏷✨

### Testing instructions
* on page load, in the "All" category, apply some filters and make sure the results contain at least one of your selected tags
* switch categories and make sure the filter tags are still applied
* in a category other than all, try updating the filters (adding, editing, clearing, applying)
* make sure when you click the `load more` button, the new resources have the correct category and filters
* make sure you cannot apply more than 10 filters
